src/react-treeview.jsx

```diff
@@ -38,9 +38,7 @@ const TreeView = React.createClass({
       <div
         {...rest}
         className={className + ' ' + arrowClassName}
-        onClick={this.handleClick}>
-        ▾
-      </div>;
```

Comment:
In JSX [all tags can be self-closing](https://facebook.github.io/react/tips/self-closing-tag.html). Although JSX may look like HTML, it isn't. It gets compiled into javascript, so it doesn't need to follow the same validation rules.